change/@fluentui-react-conformance-90ba2958-fc3c-4b00-9180-740ecb0fb58e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Not very important but this should technically be patch
```suggestion
  "type": "patch",
```

---

Oops, good catch 👍